object/virtual_device_list.go

```diff
@@ -911,6 +998,8 @@ func (l VirtualDeviceList) Type(device types.BaseVirtualDevice) string {
 		return "lsilogic-sas"
 	case *types.VirtualNVMEController:
 		return "nvme"
+	case *types.VirtualSATAController:
```

Comment:
No need for this, see `deviceNameRegexp`. Realizing we don't need the existing NVME case either.

Reply:
Want me to remove nvme or add that in a separate PR?

Reply:
Resolved in 2e77836.